src/codeu/chat/server/Server.java

```diff
@@ -195,17 +208,17 @@ public void run() {
     });
   }
 
+  //handles all server requests
```

Comment:
Similar to before, this comment isn't very useful. Consider adding detail, javadoc, or remove altogether.